biz.aQute.bndlib/src/aQute/bnd/build/Project.java

```diff
@@ -1146,7 +1146,7 @@ private URI releaseRepo(RepositoryPlugin releaseRepo, Processor context, String
 		}
 	}
 
-	private List<RepositoryPlugin> getReleaseRepos(String names) {
+	public List<RepositoryPlugin> getReleaseRepos(String names) {
```

Comment:
I don't think this needs to be public, just not private. So use default (package-private access).